ibmcloud/README.md

```diff
@@ -444,4 +454,25 @@ groups, subnet and gateway with:
 ```bash
 $ cd ibmcloud/terraform/common
 $ terraform destroy
-```
\ No newline at end of file
+```
+
+## Troubleshooting
+
+### Cluster creation when switching use `ssh_pub_key`
+
+If during the process of setting up the cluster you may experience the following error regarding your SSH key
```

Comment:
I'd combine the first two paragraphs like this:

If you provide your SSH public key via the `ssh_pub_key` Terraform variable and, having already run `terraform apply`, you run `terraform apply` again then you may see the following error regarding your SSH key:

> `Error: Error deleting SSH Key : SSH-key still in use`